equistore-core/include/equistore.hpp

```diff
@@ -1210,7 +1090,7 @@ class TensorBlock final {
 
     /// Get a view in the values in this block
     NDArray<double> values() {
```

Comment:
No, we only support `double` to get data through the C API. #56 would allow using other types.

The Python API does not go through the same mechanism to get the arrays back, so it is able to pass other data types.

Reply:
Ok then cool